packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.controller.js

```diff
@@ -7,12 +7,14 @@ import isEmpty from 'lodash/isEmpty';
 
 export default class FlavorsListController {
   /* @ngInject */
-  constructor($q, PciProjectFlavors) {
+  constructor($q, $state, PciProjectFlavors) {
     this.$q = $q;
+    this.$state = $state;
     this.PciProjectFlavors = PciProjectFlavors;
   }
 
   $onInit() {
+    this.quotaUrl = this.$state.href('pci.projects.project.quota');
```

Comment:
Thought: This should be a binding to be able to reuse the component in another context